libraries/stdlib/jvm/test/text/StringJVMTest.kt

```diff
@@ -75,53 +75,67 @@ class StringJVMTest {
     }
 
     @Test fun capitalize() {
+        fun testCapitalize(expected: String, string: String) {
+            assertEquals(expected, string.capitalize())
+            assertEquals(expected, string.replaceFirstChar { if (it.isLowerCase()) it.titlecase() else it.toString() })
+        }
         // Case mapping that results in multiple characters (validating Character.toUpperCase was not used).
         assertEquals("SSßß", "ßßß".capitalize())
+        assertEquals("Ssßß", "ßßß".replaceFirstChar { if (it.isLowerCase()) it.titlecase() else it.toString() })
```

Comment:
Note that with `uppercase[0] + uppercase(1).lowercase()` capitalize behavior changes a bit